bang/utils.py

```diff
@@ -65,6 +66,15 @@ def rarity_to_stars_images(rarity):
         un='' if rarity in models.Card.TRAINABLE_RARITIES else 'un',
     ) * rarity
 
+def generateDifficulty(difficulty, space):
+    note_image = staticImageURL('note.png')
+    val = mark_safe(u'{big_images}{small_images}'.format(
+    big_images=(u'<img src="{}" class="song-big-note">'.format(note_image) * (difficulty // 5)),
+    small_images=(u'<img src="{}" class="song-small-note">'.format(note_image) * (difficulty % 5)),))
```

Comment:
You need to indent with 4 extra spaces before these 2 lines.
Put the last `)` on a new line (because the `(` doesn't have anything after).

Reply:
✅